cli/organization.go

```diff
@@ -18,11 +18,10 @@ import (
 
 func (r *RootCmd) organizations() *serpent.Command {
 	cmd := &serpent.Command{
-		Annotations: workspaceCommand,
```

Comment:
Unsure why this annotation was here. Command is hidden, so this changes nothing from today.